.ci/validation.sh

```diff
@@ -438,15 +438,15 @@ check-since-version)
   # Travis merges the PR commit into origin/master
   # This identifies the PR's original commit
   # if it notices a merge commit
-  HEAD=`git rev-parse HEAD`
+  HEAD=$(git rev-parse HEAD)
   if git show --summary HEAD | grep ^Merge: ; then
       echo "Merge detected."
-      HEAD=`git log -n 1 --no-merges --pretty=format:"%H"`
+      HEAD=$(git log -n 1 --no-merges --pretty=format:"%H")
   fi
   # Identify previous commit to know how much to examine
   # Script assumes we are only working with 1 commit if we are in master
   # Otherwise, it looks for the common ancestor with master
-  COMMIT=`git rev-parse "$HEAD"`
+   COMMIT=$(git rev-parse "$HEAD")
```

Comment:
Please remove preceding whitespace

Reply:
@Rahulkhinchi03 please carefully look through your commits, removal is not in same commit as update

Reply:
Changes Done